homeassistant/components/twinkly/light.py

```diff
@@ -77,7 +77,7 @@ def __init__(
             self._attr_supported_color_modes = {ColorMode.RGBW}
             self._attr_color_mode = ColorMode.RGBW
             self._attr_rgbw_color = (255, 255, 255, 0)
-        elif device_info.get(DEV_LED_PROFILE) == DEV_PROFILE_RGB:
+        elif device_info.get(DEV_LED_PROFILE) in (DEV_PROFILE_RGB):
```

Comment:
This indicates that you're checking for characters in a string; is that accurate? Does this statement mean that `device_info[DEV_LED_PROFILE]` will be one of `A` or `W`?

Reply:
I was using it to search a string in a tuple. I need to revert it to the previous version, but should be working anyways. Isn’t?

Reply:
Python idiosyncracy: `(DEV_PROFILE_RGB)` isn't a tuple.

```
>>> DEV_PROFILE_RGB = "AWW"
>>> search = (DEV_PROFILE_RGB)
>>> type(search)
<class 'str'>
```

If you're wanting a tuple that contains only that string, add a comma: `(DEV_PROFILE_RGB,)`.

```
>>> DEV_PROFILE_RGB = "AWW"
>>> search = (DEV_PROFILE_RGB,)
>>> type(search)
<class 'tuple'>
```

Reply:
Fixed